browser/brave_browser_process_impl.cc

```diff
@@ -130,6 +127,9 @@ BraveBrowserProcessImpl::BraveBrowserProcessImpl(StartupData* startup_data)
   histogram_braveizer_ = brave::HistogramsBraveizer::Create();
 #endif  // BUILDFLAG(BRAVE_P3A_ENABLED)
 
+  // initialize ads stats helper
+  ads_brave_stats_helper();
```

Comment:
are you sure you want to change the order of initialization?

Reply:
nope, that was a mistake i made during a rebase. thanks for pointing that out